tmt/steps/provision/testcloud.py

```diff
@@ -484,6 +487,7 @@ def prepare_ssh_key(self, key_type: Optional[str] = None) -> None:
         if key_type is not None:
             command.extend(["-t", key_type])
         self.run(command)
+        self.verbose('key', '\n'.join(self.key), 'green')
```

Comment:
Few lines above `self.key` is initialized and ssh-keygen is called for `self.key[0]` file so we know it will be list with a single item. I'd say no need to call 'join' but to do `self.key[0]` instead (or create new variable and use it in ssh-keygen call as well).

Reply:
Sure, `self.key[0]` is used instead.